workflow/scripts/plot_isoforms.py

```
@@ -0,0 +1,23 @@
+import sys
+import os
+import pandas as pd
+
+sys.stderr = sys.stdout = open(snakemake.log[0], "w")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)